components/Bookmarks/Page.js

```diff
@@ -11,13 +11,164 @@ import {
   fontStyles,
   Center,
   Interaction,
-  linkRule
+  linkRule,
+  plainButtonRule
 } from '@project-r/styleguide'
 import { Link } from '../../lib/routes'
 import { MdBookmarkBorder } from 'react-icons/md'
 
 import { getBookmarkedDocuments } from './queries'
 
+const getConnection = data => {
+  return {
+    ...data.me.collectionItems,
+    // [TODO] deduplicate connection, not working yet
+    nodes: data.me.collectionItems.nodes.filter(
+      (elem, index, array) =>
+        index === array.findIndex(item => item.id === elem.id)
+    )
+  }
+}
+
+const mergeConnection = (data, connection) => {
+  return {
+    ...data,
+    me: {
+      ...data.me,
+      collectionItems: {
+        ...data.me.collectionItems,
+        collectionItems: connection
+      }
+    }
+  }
+}
+
+const bookmarkIcon = <MdBookmarkBorder size={22} key='icon' />
+
+const Page = ({ t }) => {
+  const [filter, setFilter] = useState('continue')
+  const [variables, setVariables] = useState({
+    collections: ['progress', 'bookmarks'],
+    progress: 'UNFINISHED'
+  })
+
+  const handleFilterClick = passedFilter => {
+    switch (passedFilter) {
+      case 'bookmarks':
+        setVariables({
+          collections: ['bookmarks'],
+          progress: 'UNFINISHED'
+        })
+        setFilter('bookmarks')
+        break
+
+      case 'read':
+        setVariables({
+          collections: ['progress', 'bookmarks'],
+          progress: 'FINISHED'
+        })
+        setFilter('read')
+        break
+
+      default:
+        setVariables({
+          collections: ['progress', 'bookmarks'],
+          progress: 'UNFINISHED'
+        })
+        setFilter('continue')
+        break
+    }
+  }
+
+  return (
+    <Frame
+      meta={{
+        title: t('nav/bookmarks')
+      }}
+      raw
+    >
+      <Center>
+        <div {...styles.title}>{t('pages/bookmarks/title')}</div>
+        <div {...styles.filter}>
+          <button
+            onClick={() => handleFilterClick('continue')}
+            {...plainButtonRule}
+            {...styles.filterItem}
+          >
+            <Interaction.H3
+              title='Weiterlesen'
+              {...styles.fiterItemText}
+              style={{
+                fontWeight: filter === 'continue' ? 'bold' : 'normal'
```

Comment:
We need to always use `fontStyles` instead of setting `fontWeight` directly (because familyName, fontWeight and fontStyle all need to align to work properly). Also currently we don't habe sans serif bold, only medium. But can we align it with debates and search where we use underline? (see «Video» and «Beliebteste»)
<img width="731" alt="Screenshot 2020-09-30 at 15 57 56" src="https://user-images.githubusercontent.com/410211/94695047-bdf48200-0335-11eb-9112-28fca3ffdd65.png">

<img width="753" alt="Screenshot 2020-09-30 at 15 57 35" src="https://user-images.githubusercontent.com/410211/94695035-baf99180-0335-11eb-9376-c9cf137d0dc8.png">

Reply:
done

Reply:
![Screenshot 2020-09-30 at 16 03 53](https://user-images.githubusercontent.com/20746301/94695727-905c0880-0336-11eb-89f6-70eda2a1bfc9.jpg)